public/app/core/components/PageNew/Page.tsx

```diff
@@ -29,10 +31,20 @@ export const Page: PageType = ({
   scrollRef,
   ...otherProps
 }) => {
+  const theme = useTheme2();
   const styles = useStyles2(getStyles);
   const navModel = usePageNav(navId, oldNavProp);
   const { chrome } = useGrafana();
 
+  const mediaQuery = window.matchMedia(`(max-width: ${theme.breakpoints.values.lg}px)`);
+  mediaQuery.addEventListener('change', (e) => setNavExpanded(e.matches ? false : navExpandedPreference));
```

Comment:
Should be a way to do this normal css media query I think (ie no js code on size change) 

This looks like it adds event listeners on every render?

Reply:
yikes, very good point about the event listeners, fixed that

i'm not sure the same behaviour is possible with just css media queries 🤔 couldn't make it work today, but i'll try again tomorrow 👍